sdk/examples/client/ledger_nano_transaction.rs

```diff
@@ -26,11 +27,13 @@ async fn main() -> Result<()> {
     let secret_manager = SecretManager::LedgerNano(LedgerSecretManager::new(true));
 
     // Generate addresses with custom account index and range
-    let addresses = client
-        .get_addresses(&secret_manager)
-        .with_account_index(0)
-        .with_range(0..2)
-        .finish()
+    let addresses = secret_manager
+        .generate_ed25519_addresses(
+            GetAddressesOptions::from_client(&client)
+                .await?
+                .with_account_index(0)
```

Comment:
Just curious, wouldn't that be the default?

Reply:
It is